cloudferrylib/os/network/neutron.py

```diff
@@ -950,9 +950,12 @@ def allocate_floatingips(self, ext_net_id):
             while True:
                 self.neutron_client.create_floatingip({
                     'floatingip': {'floating_network_id': ext_net_id}})
-        except IpAddressGenerationFailureClient:
-            LOG.info("| Floating IPs "
-                     "were allocated in network %s" % ext_net_id)
+        except neutron_exc.NeutronClientException as e:
+            if e.status_code == 409:  # 409 - Conflict
+                LOG.info("| Floating IPs were allocated in network %s" %
+                         ext_net_id)
+            else:
+                raise
```

Comment:
@oleksii-shyman Why 'raise' is bad practice? It will just re-raise actual (the last one) exception and we can find out what the problem is. But with your variant we will make debug process much more harder I guess.